### PR TITLE
Refactor values.proto for projections

### PR DESCRIPTION
**Pull request by PaulAgeyev**
Added docs for messages.

### CODE REVIEW

File: model/src/main/proto/javaclasses/exlibris/q/values.proto [hunk 1]
```diff
@@ -35,53 +35,50 @@ import "spine/net/url.proto";
 import "javaclasses/exlibris/identifiers.proto";
 import "javaclasses/exlibris/values.proto";
 
-// Consists of main book’s information.
+// Information about book.
```
Comment (alexanderkarpetc) [inline model/src/main/proto/javaclasses/exlibris/q/values.proto]: about the

File: model/src/main/proto/javaclasses/exlibris/q/values.proto [hunk 2]
```diff
@@ -35,53 +35,50 @@ import "spine/net/url.proto";
...
 
-    // The book’s title.
+    // Book title.
     BookTitle title = 2;
 
     // Categories of the book.
     repeated Category categories = 3;
 
-    // todo: docs
+    // List of the author's full name.
```
Comment (alexanderkarpetc) [inline model/src/main/proto/javaclasses/exlibris/q/values.proto]: of authors full names

File: model/src/main/proto/javaclasses/exlibris/q/values.proto [hunk 3]
```diff
@@ -35,53 +35,50 @@ import "spine/net/url.proto";
...
-    // The book can has one of the next status:
+    // The book can has one of the following status:
     //
     // 1. Borrowed — when book is borrowed by user.
     // 2. Reserved — when book is reserved by user.
     // 3. Available — when book is available to borrow or reserve.
     // 4. Expected soon — when book is not available now, but user can reserve it.
-    // 5. Overdue — when book was borrowed by user but still doesn’t returned.
-    //
+    // 5. Overdue — when book was borrowed by user but still isn’t returned.
```
Comment (alexanderkarpetc) [inline model/src/main/proto/javaclasses/exlibris/q/values.proto]: isn't -> not

File: model/src/main/proto/javaclasses/exlibris/q/values.proto [hunk 4]
```diff
@@ -35,53 +35,50 @@ import "spine/net/url.proto";
...
     oneof book_status {
         bool borrowed = 5;
         bool reserved = 6;
         bool available = 7;
         bool expected_soon = 8;
         bool overdue = 9;
     }
-
```
Comment (alexanderkarpetc) [inline model/src/main/proto/javaclasses/exlibris/q/values.proto:69]: delete whitespaces

File: model/src/main/proto/javaclasses/exlibris/q/values.proto [hunk 5]
```diff
@@ -35,53 +35,50 @@ import "spine/net/url.proto";
...
-
 }
 
-// View with information about book’s loan.
+// Information about loan of the book.
```
Comment (alexanderkarpetc) [inline model/src/main/proto/javaclasses/exlibris/q/values.proto]: about the loan

File: model/src/main/proto/javaclasses/exlibris/q/values.proto [hunk 6]
```diff
@@ -35,53 +35,50 @@ import "spine/net/url.proto";
...
 //
-// View consists of fields with inventory item,
+// Consists of fields with inventory item,
 // users who borrowed the book and their taken on and due on dates.
 //
 message LoanDetails {
 
-    // An identifier of a book. Contains ISBN base62 and number that inputted by librarian.
+    // An identifier of a book.
```
Comment (alexanderkarpetc) [inline model/src/main/proto/javaclasses/exlibris/q/values.proto:78]: a-> the

File: model/src/main/proto/javaclasses/exlibris/q/values.proto [hunk 7]
```diff
@@ -91,28 +88,27 @@ message LoanDetails {
     google.protobuf.Timestamp when_due = 4;
 }
 
-// Details about inventory.
+// Detailed information about the inventory.
 //
-// View consists of fields with book title, authors,
-// available books, amount of overdue books.
+// View consists of fields with book title, authors, book location.
```
Comment (alexanderkarpetc) [inline model/src/main/proto/javaclasses/exlibris/q/values.proto]: delete of fields with

File: model/src/main/proto/javaclasses/exlibris/q/values.proto [hunk 8]
```diff
@@ -91,28 +88,27 @@ message LoanDetails {
...
 //
 message InventoryDetails {
 
-    // book title.
+    // Book title.
     BookTitle book_title = 1;
 
-    // todo : dima
+    // List of the author's full name.
```
Comment (alexanderkarpetc) [inline model/src/main/proto/javaclasses/exlibris/q/values.proto]: same as before

File: model/src/main/proto/javaclasses/exlibris/q/values.proto [hunk 9]
```diff
@@ -155,18 +151,17 @@ message InventoryItemRecord {
...
 //
-// View consists of fields with inventory item,
-// users who borrowed the book and their taken on and due on dates.
+// Consists of fields with inventory item,
+// users who borrowed the book and their taken on, due on, and returned on dates.
 //
 message LoanDetailsHistory {
 
-    // An identifier of a book. Contains ISBN base62 and number that inputted by librarian.
+    // An identifier of a book.
```
Comment (alexanderkarpetc) [inline model/src/main/proto/javaclasses/exlibris/q/values.proto]: a -> the

File: model/src/main/proto/javaclasses/exlibris/q/values.proto [hunk 10]
```diff
@@ -189,7 +184,7 @@ message LostBook {
     // A title of the book that was lost.
     BookTitle book_title = 2;
 
-    // The list of authors.
+    // List of the author's full name.
```
Comment (alexanderkarpetc) [inline model/src/main/proto/javaclasses/exlibris/q/values.proto]: same

**Review by alexanderkarpetc — CHANGES_REQUESTED** [review]
Fix mistakes, please

File: model/src/main/proto/javaclasses/exlibris/q/values.proto [hunk 11]
```diff
@@ -91,28 +88,27 @@ message LoanDetails {
     google.protobuf.Timestamp when_due = 4;
```
Comment (Dmytry-Dyachenko) [inline model/src/main/proto/javaclasses/exlibris/q/values.proto:88]: When the book was borrowed
When the book should be returned

File: model/src/main/proto/javaclasses/exlibris/q/values.proto [hunk 12]
```diff
@@ -91,28 +88,27 @@ message LoanDetails {
...
 
-    // List with information about books.
+    // List of information about the location of books.
     repeated InventoryItem items = 3;
 }
 
-//
+// Information about the location of books.
 //
 message InventoryItem {
```
Comment (Dmytry-Dyachenko) [inline model/src/main/proto/javaclasses/exlibris/q/values.proto]: I think that the message that has information about the location of the item, should consist it in its name.

File: model/src/main/proto/javaclasses/exlibris/q/values.proto [hunk 13]
```diff
@@ -131,16 +127,16 @@ message ReservationDetails {
     google.protobuf.Timestamp made_on = 2;
```
Comment (Dmytry-Dyachenko) [inline model/src/main/proto/javaclasses/exlibris/q/values.proto:127]: line 123:
who reserved the book

File: model/src/main/proto/javaclasses/exlibris/q/values.proto [hunk 14]
```diff
@@ -131,16 +127,16 @@ message ReservationDetails {
...
 
-// View with information about inventory item record.
+// Information about inventory item record.
 //
-// View consists of fields about inventory item, actions and action date.
+// Consists of fields about inventory item, actions and action date.
 //
 message InventoryItemRecord {
 
     // An identifier is represented by book id and item number.
```
Comment (Dmytry-Dyachenko) [inline model/src/main/proto/javaclasses/exlibris/q/values.proto]: by the book ID or
by the book identifier

**Review by Dmytry-Dyachenko — CHANGES_REQUESTED** [review]
Fix all mistakes, please

**Review by alexanderkarpetc — APPROVED** [review]
LGTM

**Review by Dmytry-Dyachenko — APPROVED** [review]
LGTM

**Pull request merged**